### ISSUE TITLE
Atmega 8 doesn't working

### ISSUE THREAD

**Issue by evgen05** [leomil72/analogComp#4]
on Arduino NG whith atmega 8 this library doesn't working.

**Comment by leomil72** [comment]
Should compile and work. What's the problem, exactly? Does it stop during compiling, or doens't work as expected?

**Comment by evgen05** [comment]
It's compiling whithout mistakes, but doesn't work. The same code compiling and working on atmega 328.

**Comment by leomil72** [comment]
I don't have an Atmega8 anymore so at the moment I'm not able to investigate this issue... Which function did you try? Which pins did you use? Please provide the more informations you can. Thanks

**Comment by evgen05** [comment]
I try to use sketch from example analogcomp_Enableinterrupt. Pins use 12 and 13 on chip(6 and 7 on board). Try use exporting hex in proteus - the same thing, doesn't working.

**Comment by evgen05** [comment]
I try to use sketch from example analogcomp_Enableinterrupt. Pins use 12 and 13 on chip(6 and 7 on board). Try use exporting hex in proteus - the same thing, doesn't working.

**Comment by leomil72** [comment]
I'm not sure if this this the cause, but I've seen that the boards.txt file of newer versions of the Arduino IDE show a different definition of the MCU used on the Arduino NG. If you look into that file, I'll find that there are a couple more of boards definitions that are commented, and one is with the old Atmega8.
So you could try 2 solutions:
1) comment the whole definition of the board "Arduino NG or older" and uncomment the one named "Arduino NG with Atmega8" or
2) try adding this line:
#define __AVR_ATmega8__
just before the inclusion of the analogComp library.

Please provide me your feedback. Thank you

**Comment by evgen05** [comment]
yes, i've choose atmega8 in arduino NG. Arduino NG was with atmega 168 and atmega 8.
